NuGet.config

```diff
@@ -4,17 +4,14 @@
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
     <!--  Begin: Package sources from dotnet-aspnetcore -->
+    <add key="darc-pub-dotnet-aspnetcore-3083783" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-aspnetcore-30837836/nuget/v3/index.json" />
```

Comment:
Do we still need this feed? Users should be able to get all these packages from nuget.org

Reply:
```suggestion
```